src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -487,6 +487,10 @@
     text: az acr repository update -n MyRegistry --image hello-world:latest --write-enabled false
   - name: Update the attributes of the image referenced by digest 'hello-world@sha256:abc123' to disable write operation.
     text: az acr repository update -n MyRegistry --image hello-world@sha256:abc123 --write-enabled false
+  - name: Update the attributes of the Repository to disable Teleport feature.
+    text: az acr repository update -n MyRegistry --teleport-enabled false
+  - name: Update the attributes to enable Teleport feature on all new images in the Repository, which improves container startup performance.
```

Comment:
I am not sure about the phrasing here because it sounds like by enabling the flag container startup performance is automatically improved which is not the case always. That's only true when teleport is enabled and the customer runs on a teleport client. Id probably rewrite it as: 

Update the attributes of the Repository to enable Teleport feature on new images pushed to the Repository. Enabling this improves container startup performance on teleport configured clients.

Out of curiosity what happens if we mark something as Teleport Enabled, push some images and then briefly set it to disabled, after which we enable it again, are the previously enabled images expected to be teleport enabled?

Reply:
Thanks @esteban, I used this description from the swagger

https://github.com/shahzzzam/azure-rest-api-specs/blob/c9220138814fd21766c643c8dcf62d976bb83034/specification/containerregistry/data-plane/Microsoft.ContainerRegistry/preview/2019-08-15/containerregistry.json#L2082

I kept the same for consistency purposes. 

Your description is definitely more accurate that it should only enhance perf on teleport-enabled clients. I will use the term "teleport-enabled hosts" instead of teleport configured clients

Reply:
re: enable disable
we delete the shares on disable operation. So once you enable it, it will only teleport new images.